docs/source/Participate/PlatformIO.rst

```diff
@@ -124,18 +126,19 @@ The environment definitions all have at least the used micro controller in the n
 
 For example:
 
-* ..._ESP8266_4M -> ESP8266 has external flash, which can vary in size from 512 kB to 16 MB.
+* ..._ESP8266_4Mnn -> ESP8266 has external flash, which can vary in size from 512 kB to 16 MB, with nn configured as filesystem.
 * ..._ESP8285_1M -> ESP8285 has the flash internal, so is always 1 MB.
 * ..._ESP32_4M316k -> ESP32 with 4 MB flash and a 1.8 MB partition for the sketch. (316k SPIFFS)
+* ..._ESP32_16M2M -> ESP32 with 16 MB flash and a 4 MB partition for the sketch. (2MB LittleFS)
 
 Make a custom build using PlatformIO
 ------------------------------------
 
-The easiest is to go for the environment "env:custom_ESP8266_4M" and unfold that one.
+The easiest is to go for the environment "custom_ESP8266_4M1M" and unfold that one.
 Then select "Build" to see if it will start building.
 
-If that's working, you can open the file "pre_extra_script.py" and add or remove the plugins and controllers you need.
-That Python file is used in the "env:custom_ESP8266_4M" to define what should be embedded and what not.
+If that's working, you can open the file "pre_custom_esp8266.py" and add or remove the plugins and controllers you need.
+That Python file is used in the "env:custom_ESP8266_4M1M" (or any "custom" build environment) to define what should be embedded and what not.
```

Comment:
Maybe also describe the option to rename `Custom-sample.h` to `Custom.h`.
This `Custom.h` (with capital C) will be used in the build if the file is present.

Also this file will not be offered to be added to a commit in Git, nor will it ever be overwritten if you pull new commits from GitHub and thus may be a more practical way to define your own builds.

Reply:
I'll add that.